docs/griptape-framework/data/vector-store-drivers.md

```diff
@@ -148,40 +151,44 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
 11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
 
 ## MongoDbAtlasVectorStoreDriver
+
 This driver provides support for storing vector data in a MongoDB Atlas database. The driver includes methods for performing standard vector operations such as inserting, updating, querying, and loading vectors.
 
 Below is a detailed example of how the MongoDbAtlasVectorStoreDriver can be used to interact with a MongoDB Atlas instance:
 
 ```python
 from griptape.drivers import MongoDbAtlasVectorStoreDriver
+from griptape.loaders import WebLoader
+import os
 
+host = os.getenv("MONGODB_HOST")
+username = os.getenv("MONGODB_USERNAME")
+password = os.getenv("MONGODB_PASSWORD")
+database_name = os.getenv("MONGODB_DATABASE_NAME")
+collection_name = os.getenv("MONGODB_COLLECTION_NAME")
 # Initialize the vector store driver
 vector_store = MongoDbAtlasVectorStoreDriver(
-    connection_string="mongodb+srv://<username>:<password>@cluster.mongodb.net/<dbname>",
-    database_name="myDatabase",
-    collection_name="myCollection"
+    connection_string=f"mongodb+srv://{username}:{password}@{host}/{database_name}",
+    database_name=database_name,
+    collection_name=collection_name,
 )
 
-# Upsert a vector
-vector_id = vector_store.upsert_vector(
-    vector=[0.1, 0.2, 0.3],
-    namespace="myNamespace",
-    meta={"info": "sample"}
-)
-
-# Load a specific entry by vector ID
-entry = vector_store.load_entry(vector_id=vector_id)
-print(entry)
+# Load artifacts from the web
+artifacts = WebLoader(max_tokens=200).load("https://www.griptape.ai")
 
-# Query the collection
-results = vector_store.query(
-    query="What is griptape?",
-    count=5
+# Upsert the artifacts into the vector store
+vector_store.upsert_text_artifacts(
+    {
+        "griptape": artifacts,
+    }
 )
-print(results)
+
+result = vector_store.query(query="What is griptape?")
+print(result)
```

Comment:
I can't remember if we decided to remove `print`s from all examples...

Reply:
@vasinov eventually might move this to a REPL example, I'm making a note to improve in the futrue.